package.json

```diff
@@ -2,8 +2,9 @@
   "name": "lokinet-gui",
   "productName": "Lokinet-GUI",
   "description": "A Control panel for Lokinet",
-  "version": "0.9.9",
+  "version": "1.0.0",
```

Comment:
shouldnt this be `0.9.10`?

Reply:
No; this is just the gui version, not the underlying lokinet version (the GUI reports the lokinet version that it gets from RPC more prominently anyway, so there's no reason they need to match).

Reply:
(Also even if we try to match them, I am sure they would deviate over time).